.github/dependabot.yml

```diff
@@ -2,11 +2,14 @@
 
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "/"
+  - package-ecosystem: 'npm'
+    directory: '/'
     schedule:
-      interval: "weekly"
+      interval: 'weekly'
     pull-request-branch-name:
       # Separate sections of the branch name with a hyphen because docker hub doesn't want slashes
       # for example, `dependabot-npm_and_yarn-next_js-acorn-6.4.1`
-      separator: "-"
+      separator: '-'
+    versioning-strategy: increase-if-necessary
+    open-pull-requests-limit: 10
```

Comment:
I set this explicitly to only 2 dependabot PRs open at a time. https://github.com/UserOfficeProject/user-office-core/pull/354
It was needed because when too many PRs are open at the same time we run out of available job workers and it is hard to run our PR actions.